python/GafferCortexUI/ParameterisedHolderUI.py

```diff
@@ -129,8 +129,13 @@ def getToolTip( self ) :
 
 	def __infoText( self ) :
 
-		result = Gaffer.Metadata.nodeDescription( self.__node )
-		summary = Gaffer.Metadata.nodeValue( self.__node, "summary" )
+		## \todo: NodeUI should provide setContext()/getContext() methods
+		## and we should use those to get the proper context here.
+		context = node.scriptNode().context() if node.scriptNode() else Gaffer.Context.current()
```

Comment:
Should be __node rather than node here I think - I'm getting "NameError: global name 'node' is not defined".